### PR TITLE
renaming and deprecating get functions and removing league metadata f…

### PR DESCRIPTION
**Pull request by JaseZiv**
renaming and deprecating get functions and removing league metadata from match level functions

### CODE REVIEW

**Comment by JaseZiv** [comment]
This is a fairly big update... In summary:

**New functions (and deprecated old ones)**

* `fb_advanced_match_stats()` replaces `get_advanced_match_stats`
* `fb_match_lineups()` replaces `get_match_lineups`
* `fb_match_results()` replaces `get_match_results`
* `fb_match_report()` replaces `get_match_report`
* `fb_match_shooting()` replaces `get_match_shooting`
* `fb_match_summary()` replaces `get_match_summary`
* `fb_match_urls()` replaces `get_match_urls`
* `fb_season_team_stats()` replaces `get_season_team_stats`
* `fb_team_match_results()` replaces `get_team_match_results`
* `tm_player_market_values()` replaces `get_player_market_values`
* `tm_player_transfer_history` replaces `player_transfer_history`

**Improvements**

The following functions now no longer return league/season metadata, including columns `League`, `Gender`, `Country`, `Season`:

* `fb_match_summary()`
* `fb_advanced_match_stats()`
* `fb_match_report()`

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: worldfootballR
 Title: Extract and Clean World Football (Soccer) Data
-Version: 0.5.10.1000
+Version: 0.5.2
```
Comment (tonyelhabr) [inline DESCRIPTION:4]: shouldn't this be `5.11` or `5.12`? we've already had a `5.2`

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
+# worldfootballR 0.5.2
```
Comment (tonyelhabr) [inline NEWS.md:1]: see DESCRIPTION note

**Review by tonyelhabr — APPROVED** [review]
lgtm aside from the package version. i've wanted these functions to be renamed for a while 😅

File: DESCRIPTION (hunk 1 above)
Comment (JaseZiv) [reply DESCRIPTION:4]: How embarrassing 😳

Thanks for picking that up!

**Pull request merged**